### ISSUE TITLE
100% CPU usage and slow input response when Windows spellchecking is enabled

### ISSUE THREAD

**Issue by grandsilence** [telegramdesktop/tdesktop#8833]
### Steps to reproduce
1. Open settings and enable system spellchecking
2. Open any chat tab
3. Write a lot of symbols in chat textbox like:
```
......................................................................................................................................................................................................................................
```

### Expected behaviour
Spellchecking method should be called only once, 200 ms after the last input, not on every key press (debounce event via timer required).

### Actual behaviour
On each symbol type Telegram calling spellcheck method and load CPU on each key press.
![cpu](https://user-images.githubusercontent.com/9817522/96152026-bd064780-0f14-11eb-8a68-f1df6eeca92c.png)

### Configuration
**Operating system:** Windows 10 1803
**Version of Telegram Desktop:** 2.4.3
**Used theme**: Default

**Comment by ilya-fedin** [comment]
Are you sure that this is due to the spellchecker? Telegram writes drafts synchronously to disk and that may cause the same behavior, there are an issue for that already:#7952

**Comment by grandsilence** [comment]
@ilya-fedin Yes, I'm sure. If I turn off grammar checking, CPU consumption is **0.1%** for `Telegram.exe`. I just checked it out.

**Comment by grandsilence** [comment]
Looks like #8612 already reported about it.

**Comment by ilya-fedin** [comment]
> 200 ms after the last input

Just wondering, why 200ms exactly?

**Comment by grandsilence** [comment]
@ilya-fedin 100-200ms is average value of user reaction. So it's a great value for debounce / throttling events.

**Comment by ilya-fedin** [comment]
Thanks.

**Comment by 23rd** [comment]
Thanks for your report, I'll check it later.

**Comment by 23rd** [comment]
@grandsilence Are you sure that pure dots without any letter are the cause of high CPU usage?

**Comment by grandsilence** [comment]
@23rd Yes, I’m sure. Just dots cause of 100% cpu usage. I can make a GIF.

**Comment by 23rd** [comment]
> I can make a GIF.

I would appreciate it.

**Comment by grandsilence** [comment]
@23rd GIFs:

# Spellchecking ON

![Spellchecking is enabled](https://user-images.githubusercontent.com/9817522/96904352-a9705900-149f-11eb-89df-8c9a1db36124.gif)

# Spellchecking OFF

![Spellchecking is disabled](https://user-images.githubusercontent.com/9817522/96903856-04ee1700-149f-11eb-9a70-0dec52af13b6.gif)

**Comment by grandsilence** [comment]
The problem with the CPU load **has been solved**, as of version **v2.5.9 (Beta)**.  
It remains to fix the problem with microlags during the printing of the message (issue #7952).
